shared/chats/mappers.ts

```diff
@@ -2,15 +2,24 @@ import { ILiveRoomInfo, IMessage, IRoomHeader } from "./dto";
 
 export const mapDBToIMessage = (userId: number, dbData: any): IMessage => {
 	return {
-		roomId: dbData.roomId,
+		roomId: dbData.room_id,
 		nickname: dbData.nickname,
 		message: dbData.message,
```

Comment:
@HungKungE
이미 수정됐겠지만, 이부분 `message`가 아니고 `content`로 조회되었어야 했을 것 같습니다.

```suggestion
		message: dbData.content,
```